### DISCUSSION TITLE
Change in project structure in the repo

### DISCUSSION THREAD

**Discussion by Meminseeker** [bounswe/bounswe2024group11#182] [General]
@umitcan07 After your last commit, we do have another directory named `web` in addition to the `frontend` and `mobile` directories. What is the reason for that?

**Comment by umitcan07** [comment]
@Meminseeker We needed to create a new app from scratch to incorporate [MantineUI](https://mantine.dev/getting-started/) which is our UI library. The easiest way is use a template. We use a template provided by them to avoid any of the annoying errors.

We could've sticked to the name frontend, but it just felt natural to me and @hks1444  to go with web.

**Reply by Meminseeker** [reply]
Okay. I asked to learn what you did. Also, I want to be sure this is not done by mistake.

**Comment by umitcan07** [comment]
Additionally, we should've opened a PR for it. Now creating another PR for other stuff on web

**Reply by Meminseeker** [reply]
Since we were starting to initialize the repo, we also pushed directly to the main branch. Maybe from now on, we all should use PRs.

**Reply by umitcan07** [reply]
Agree, we have already created a PR, reviewed it and merged it for web app

**Comment by Meminseeker** [comment]
Please be aware that this new directory does not have a .gitignore file. Some files we don't want to see in our repo were pushed, such as .vscode/.

**Reply by hks1444** [reply]
We had make .vscode file that way because we wanted to use Biome for formatting. This was the way to do it.

**Reply by umitcan07** [reply]
.vscode is intended to be NOT gitignored though. It is our bridge to ensure we all use same vscode settings for this repository. So that we can match our formatters etc etc.

**Reply by umitcan07** [reply]
We have added gitignore, thanks for the heads-up
